iac-extensions/kubernetes/src/main/java/org/sonar/iac/kubernetes/checks/RBACWildcardCheck.java

```diff
@@ -52,7 +52,7 @@ void registerObjectCheck() {
 
   @Override
   void initializeCheck(CheckContext ctx) {
-    if (ctx instanceof HelmAwareCheckContext helmContext) {
+    if (ctx instanceof KubernetesCheckContext helmContext) {
```

Comment:
```suggestion
    if (ctx instanceof KubernetesCheckContext kubernetesCheckContext) {
```
Maybe also rename variable?